### DISCUSSION TITLE
nacos

### DISCUSSION THREAD

**Discussion by 178397462** [zuihou/zuihou-admin-cloud#94] [01.常见问题]
启动 lamp-authority-server时，连接不上nacos，其他工程类同，有大神知道吗？

**Comment by zuihou** [comment]
https://github.com/zuihou/lamp-cloud/discussions/49

**Comment by zuihou** [comment]
https://github.com/zuihou/lamp-cloud/discussions/52

**Comment by sjfengOS** [comment]
nacos 的版本问题  一定要使用对应的版本  nacos-client 是2.0  nacos服务版本也需要是2.0   nacos-client是2.0  nacos服务也要使用2.0
